internal/output/dockerfile/step/copy.go

```diff
@@ -58,6 +58,10 @@ func (step *CopyStep) Generate(w io.Writer) error {
 		fromClause = fmt.Sprintf("--from=%s ", step.from)
 	}
 
+	if step.platform != "" {
+		fromClause = fmt.Sprintf("--platform=%s %s", step.platform, fromClause)
+	}
+
```

Comment:
should this go in the if block above? Is `platform` without `from` valid?

Reply:
yes, it should be valid copying from a stage